tidb-cloud/serverless-tier-limitations.md

```diff
@@ -7,49 +7,53 @@ summary: Learn about the limitations of TiDB Serverless.
+- No "IP Access List" support.
 
-### Transaction
+### Encryption
 
-- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase.
+- Data persisted in your TiDB Serverless cluster undergoes encryption utilizing the encryption tool provided by the cloud provider responsible for managing your cluster. However, TiDB Serverless does not provide additional optional measures for protecting data at-rest on disks beyond infrastructure-level encryption.
+- Using [customer-managed encryption keys (CMEK)](/tidb-cloud/tidb-cloud-encrypt-cmek.md) is currently unavailable.
 
-### Connection
+### Maintenance window
 
-- Only [Standard Connection](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to TiDB Serverless clusters. 
-- No "IP Access List" support.
+- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is currently unavailable.
 
-### Monitoring
+### Monitoring and Diagnosis
 
-- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently not available for TiDB Serverless.
-- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently not available for TiDB Serverless.
-- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently not available for TiDB Serverless.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently unavailable.
+- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently unavailable.
+- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently unavailable.
+- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is currently unavailable.
 
-### Diagnosis
+### Self service upgrades
 
-- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is unavailable for TiDB Serverless.
+- TiDB Serverless is a fully managed deployment of TiDB. Major and minor version upgrades of TiDB are handled by TiDB Cloud, therefore can not be initiated by users.
 
 ### Stream data
 
-* [Changefeed](/tidb-cloud/changefeed-overview.md) is not supported for TiDB Serverless currently.
-* [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is not supported for TiDB Serverless currently.
+- [Changefeed](/tidb-cloud/changefeed-overview.md) is currently unavailable.
+- [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is currently unavailable.
 
-### Maintenance window
+### Others
 
-- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is unavailable for TiDB Serverless.
+- [Time to live (TTL)](/time-to-live.md) is currently unavailable.
+- Transaction can not last longer than 30m
+- Total size of a single transaction is set to no more than XX MB
```

Comment:
Please update xx with the actual number.

Reply:
Confirmed that the limitation is gone.